### PR TITLE
[terra-framework] Removed inline style for dev-site components Part 6

### LINKED ISSUE #657: Remove inline styles in dev site components
# Feature Request

## Description

We want to remove inline styles from our dev site components to set a better example for consumers of terra.  As a replacement, we should pass in classes and use css to style the components.

### PR DESCRIPTION
**Pull request by pranav300**
Summary
Removed all inline styles and replaced the same styles with external CSS for the following packages.

1. - Hookshot
2. - Navigation Layout

Partially resolves #657

### CODE REVIEW

File: packages/terra-hookshot/src/terra-dev-site/test/common/HookshotTestDocCommon.scss [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    @include hookshot-common-content();
+    margin-left: 50px;
+  }
+
+  .hookshot-auto-90 {
+    @include hookshot-common-content();
+    margin-left: 60px;
+  }
+
+  .hookshot-auto--90 {
```
Comment (emilyrohrbough) [inline packages/terra-hookshot/src/terra-dev-site/test/common/HookshotTestDocCommon.scss]: can we change this class name to something more meaningful? `hookshot-auto-90` and `hookshot-auto--90` don't have any differentiation

File: packages/terra-navigation-layout/src/terra-dev-site/doc/example/HeaderExample.jsx [hunk 2]
```diff
@@ -1,41 +1,35 @@
...
-          type="button"
-          style={inlineButtonStyle}
-          onClick={layoutConfig.toggleMenu}
-        >
-Toggle Menu
-        </button>
+          <button
+            type="button"
+            onClick={layoutConfig.toggleMenu}
+            className={cx('button')}
```
Comment (emilyrohrbough) [inline packages/terra-navigation-layout/src/terra-dev-site/doc/example/HeaderExample.jsx]: There isn't a button class name in the HeaderExampler.scss file. Do we need this?

File: packages/terra-navigation-layout/src/terra-dev-site/doc/example/HeaderExample.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+@import './NavigationLayoutDocCommon';
```
Comment (emilyrohrbough) [inline packages/terra-navigation-layout/src/terra-dev-site/doc/example/HeaderExample.scss]: Lets add all the styles directly that are needed, or we can import `NavigationLayoutDocCommon` directly into the examples and bind those class names

File: packages/terra-hookshot/src/terra-dev-site/test/common/HookshotTestDocCommon.scss [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    margin-top: 100px;
+  }
+
+  .hookshot-auto-pushed {
+    @include hookshot-common-content();
+    margin-left: 110px;
+    margin-top: 100px;
+  }
+
+  .hookshot-wrapper-primary {
```
Comment (emilyrohrbough) [inline packages/terra-hookshot/src/terra-dev-site/test/common/HookshotTestDocCommon.scss]: I am seeing the following repeated several times throughout this file. Can we create a single class name that can be referenced to add this style? Then lets drop the @include and we can bind that class name for any element that needs this style
```diff
- @mixin hookshot-common-content() {
+ .content {
  background-color: #d3d3d3;
  height: 36px;
  margin-left: 220px;
  margin-right: 220px;
  margin-top: 55px;
- width: 100px;
}
-  .hookshot-auto--90 {
-    @include hookshot-common-wrapper();
-  }

-  .hookshot-wrapper-auto-flipped {
-    @include hookshot-common-wrapper();
-  }
```

File: packages/terra-hookshot/src/terra-dev-site/test/common/HookshotTestDocCommon.scss (hunk 1 above)
Comment (pranav300) [reply packages/terra-hookshot/src/terra-dev-site/test/common/HookshotTestDocCommon.scss]: Fixed here https://github.com/cerner/terra-framework/pull/742/commits/d320272a142e98bfee8b1d612816f9fabd7aa36d.

File: packages/terra-navigation-layout/src/terra-dev-site/doc/example/HeaderExample.jsx (hunk 2 above)
Comment (pranav300) [reply packages/terra-navigation-layout/src/terra-dev-site/doc/example/HeaderExample.jsx]: Fixed here https://github.com/cerner/terra-framework/pull/742/commits/d320272a142e98bfee8b1d612816f9fabd7aa36d.

File: packages/terra-navigation-layout/src/terra-dev-site/doc/example/HeaderExample.scss (hunk 3 above)
Comment (pranav300) [reply packages/terra-navigation-layout/src/terra-dev-site/doc/example/HeaderExample.scss]: Fixed here https://github.com/cerner/terra-framework/pull/742/commits/d320272a142e98bfee8b1d612816f9fabd7aa36d.

File: packages/terra-hookshot/src/terra-dev-site/test/common/HookshotTestDocCommon.scss (hunk 4 above)
Comment (pranav300) [reply packages/terra-hookshot/src/terra-dev-site/test/common/HookshotTestDocCommon.scss]: Fixed here https://github.com/cerner/terra-framework/pull/742/commits/d320272a142e98bfee8b1d612816f9fabd7aa36d.

**Pull request merged**